pkg/components/apiserverauth/component.go

```diff
@@ -22,6 +22,20 @@ var ApiserverAuthComponent = Component{
 			{
 				IncludeAll: []string{"bz-apiserver-auth"},
 			},
+			{Suite: "Authentication"},
+			{Suite: "Group sync related scenarios"},
+			{Suite: "SCC policy related scenarios"},
+			{Suite: "Seccomp part of SCC policy should be kept and working after upgrade"},
+			{Suite: "Security_and_Compliance"},
```

Comment:
Please drop Security_and_Compliance. I don't think apiserverauth contains this keyword.